package.json

```diff
@@ -16,105 +16,106 @@
     "start:development:client": "node client/scripts/start.js"
   },
   "dependencies": {
-    "argon2": "^0.16.2",
-    "autoprefixer": "^7.1.2",
```

Comment:
One should see if the changes introduced in those versions are impacting Flood

https://github.com/postcss/autoprefixer/releases/tag/8.4.0
https://github.com/postcss/autoprefixer/releases/tag/8.3.0
https://github.com/postcss/autoprefixer/releases/tag/8.2.0
https://github.com/postcss/autoprefixer/releases/tag/8.1.0
https://github.com/postcss/autoprefixer/releases/tag/8.0.0
https://github.com/postcss/autoprefixer/releases/tag/7.2.0